ts/torch_handler/base_handler.py

```diff
@@ -64,16 +62,16 @@ def initialize(self, context):
         if model_file:
             logger.debug("Loading eager model")
             self.model = self._load_pickled_model(model_dir, model_file, model_pt_path)
+            self.model.to(self.device)
+            
         else:
             logger.debug("Loading torchscript model")
             if not os.path.isfile(model_pt_path):
                 raise RuntimeError("Missing the model.pt file")
 
             self.model = self._load_torchscript_model(model_pt_path)
 
-        self.model.to(self.device)
```

Comment:
why are we removing this?

Reply:
@vdantu we are moving it to the IF block for the eager mode in line#65 as it is not required for torchscript, in fact having the leaving it here will make the duplicated process for torchscript mode.